pkg/client/unversioned/portforward/portforward.go

```diff
@@ -250,67 +254,79 @@ func (pf *PortForwarder) waitForConnection(listener net.Listener, port Forwarded
 func (pf *PortForwarder) handleConnection(conn net.Conn, port ForwardedPort) {
 	defer conn.Close()
 
-	glog.Infof("Handling connection for %d", port.Local)
+	pf.connectionIDLock.Lock()
+	connectionID := pf.nextConnectionID
+	pf.nextConnectionID++
```

Comment:
Actually, is it worth it? Won't I have to hold the lock here, to get the connection ID, and then again in the helper method, to increment it?